packages/teams-js/src/public/secondaryBrowser.ts

```diff
@@ -6,14 +6,16 @@ import { ErrorCode } from './interfaces';
 import { runtime } from './runtime';
 
 /**
- * Namespace to power up the in-app browser experiences in the Host App.
- * For e.g., opening a URL in the Host App inside a browser
+ * Namespace to power up the in-app browser experiences in the host app.
+ * For e.g., opening a URL in the host app inside a browser
  *
  * @beta
  */
 export namespace secondaryBrowser {
   /**
-   * Open a URL in the secondary browser aka in-app browser
+   * Open a URL in the secondary browser 
```

Comment:
Line breaks in source don't necessarily render in the published doc, so these three lines will probably publish as a run-on sentence. Could you add punctuation?